src/sentry/web/urls.py

```diff
@@ -135,6 +135,13 @@ def init_all_applications():
     ),
     url(r'^api/store/schema$', api.StoreSchemaView.as_view(), name='sentry-api-store-schema'),
 
+    # Frontend client config
+    url(
+        r'^api/client-config/?$',
```

Comment:
Not 100% sure that it should NOT be under the `/0/` version. I just put it here since it seemed generic enough.

Reply:
I think it makes sense here considering it's configuration to run the app itself – not some primitive of Sentry. `/api/0/*` is more for those primitives, imo. I could probably be convinced this is one, but I think it's fine here.